packages/boiler-task-selenium/src/runner/webdriverio/run.js

```diff
@@ -30,13 +31,12 @@ export default function(opts, config, cb) {
    */
   function runCp(opt, local, concurrent) {
     const {baseUrl} = opt;
-    const message = `Starting ${concurrent && opts.length > 1 ? 'Concurrent' : 'Parallel'} Tests for [${magenta(baseUrl)}]`;
+    const message = `Starting ${concurrent && testSettings.length > 1 ? 'Concurrent' : 'Parallel'} Tests for [${magenta(baseUrl)}]`;
 
     log(message);
 
     const env = _.merge({}, process.env, {
-      // TODO: Make a better way to override wdio options dynamically
-      WDIO_CONFIG: JSON.stringify(Object.assign(opt,  {maxInstances: instances})),
+      WDIO_CONFIG: JSON.stringify(opt),
       TEST_ENV: JSON.stringify({local})
     });
 
```

Comment:
seems like a lot of this logic for logging should be shared between nigthwatch and webdriver. Since both are spawning a child process it seems that we could still keep the `spawn` file that gets the options from both nigthwatch and webdriver task and then does logging. I'd like to keep logging somewhat consistent and it's nice to have the info of what test files are being run.